public/landing-page-same-style.css

```diff
@@ -0,0 +1,276 @@
+/* Base Styles */
+body {
+  height: 100vh;
+  background-color: #000;
+  font-family: Helvetica, Arial, sans-serif;;
```

Comment:
add to body:

```
  -webkit-font-smoothing: antialiased !important;
  -moz-font-smoothing: antialiased !important;
   -o-font-smoothing: antialiased !important;
   font-smoothing: antialiased;
  text-rendering: optimizelegibility !important;
```

You ca also try:
```
-moz-osx-font-smoothing: grayscale;
  letter-spacing: .03em;
  ```
  
  An if you really want to go deeper:
 ```
/* Read here for more https://www.smashingmagazine.com/2020/07/css-techniques-legibility/ */
```